modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -102,12 +109,36 @@ export default function ListEntries() {
 				queryParams={{dataListViewId}}
 				scope={appId}
 			>
-				{buildEntries({
-					dataDefinition,
-					fieldNames,
-					permissions,
-					scope: appId,
-				})}
+				{(entry, index) => {
+					const statuses = {
+						approved: {
+							displayType: 'success',
+							label: Liferay.Language.get('approved'),
+						},
+						pending: {
+							displayType: 'info',
+							label: Liferay.Language.get('pending'),
+						},
```

Comment:
Hey, nice PR!
What do you think moving this object to a constants file so it can be reused in the future?

Reply:
Thanks Gab! I will do this on the next PR that i reuse this values on View Entry page.